tests/tensor_search/integ_tests/test_add_documents_combined.py

```diff
@@ -65,7 +65,8 @@ def test_add_documents_with_truncated_image(self):
 
         documents = [
             {
-                "image_field_1": "https://marqo-assets.s3.amazonaws.com/tests/images/ai_hippo_statue.png",
+                "image_field_1": "https://raw.githubusercontent.com/marqo-ai/marqo/mainline"
```

Comment:
What's the reason for this change?

Reply:
It is reported that this test fails occasionally. We believe it's a transient network issue. My experience is that GitHub is more reliable than s3 in unit tests.